cmd/collector/app/handler/otlp_receiver_test.go

```diff
@@ -7,32 +7,35 @@ import (
 	"context"
 	"errors"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configgrpc"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/config/confignet"
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/pdata/ptrace"
 	"go.opentelemetry.io/collector/pipeline"
 	"go.opentelemetry.io/collector/receiver"
 	"go.opentelemetry.io/collector/receiver/otlpreceiver"
 
 	"github.com/jaegertracing/jaeger/cmd/collector/app/flags"
-	"github.com/jaegertracing/jaeger/pkg/config/corscfg"
-	"github.com/jaegertracing/jaeger/pkg/config/tlscfg"
 	"github.com/jaegertracing/jaeger/pkg/tenancy"
 	"github.com/jaegertracing/jaeger/pkg/testutils"
 )
 
 func optionsWithPorts(port string) *flags.CollectorOptions {
 	opts := &flags.CollectorOptions{}
-	opts.OTLP.GRPC = flags.GRPCOptions{
-		HostPort: port,
+	opts.OTLP.HTTP = confighttp.ServerConfig{
```

Comment:
can we just define these in the line above when initializing `&flags.CollectorOptions{}`?